pom.xml

```diff
@@ -218,9 +218,9 @@
 
 
     <dependency><!-- we contribute AbstractBuildParameters for Git if it's available -->
-      <groupId>org.jvnet.hudson.plugins</groupId>
+      <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>parameterized-trigger</artifactId>
-      <version>2.4</version>
+      <version>2.9</version>
```

Comment:
2.33 is the latest version and it is compatible with 1.596.1 which is lower than our current core. @MarkEWaite I'll leave it as an exercise to you to test this version